### PR TITLE
Weld + Dialog documentation

### PR DESCRIPTION
**Pull request by Erazias**
I've finally figured out how to make multiple commits in one pull requests, so here is the documentation for welds and dialogs along with objects-index.md being updated to match them.

### CODE REVIEW

File: objects/instances/dialog.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+#### GoodbyeDialog (String)
+
+* [x] Editable without script
+* [ ] Read only
+
+```
+```
+
```
Comment (FloofyPlasma) [inline objects/instances/dialog.md:70]: You are missing the InUse property.

File: objects/instances/weld.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+```
+workspace
+```
+
+#### Part0 (String)
+
+* [ ] Editable without script
+* [x] Read only
+
```
Comment (FloofyPlasma) [inline objects/instances/weld.md:76]: Part0 is not read only.

File: objects/instances/weld.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+```
+nil
+```
+
+#### Part1 (String)
+
+* [x] Editable without script
```
Comment (FloofyPlasma) [inline objects/instances/weld.md:83]: Part1 is not editable without a script.

**Review by FloofyPlasma — CHANGES_REQUESTED** [review]
Please fix the issues I pointed out and add your pages to SUMMARY.md in the root folder of the repo.

**Comment by Erazias** [comment]
Done.

**Comment by Erazias** [comment]
I will make a new pull request fixing all of this, along with adding new instances.

**Pull request closed without merging**